README.md

```diff
@@ -86,7 +86,6 @@ To make use of this project on the console, kindly run `ruby main.rb`
 
```

Comment:
![image](https://github.com/eamador1/school-library/assets/93039359/47adb59f-45bf-4f69-a4a0-7464a42dfedf)

- [ ] It would be nice if you fix the LinkedIn link in your readme file which you can see in the image above :eyes: because right now it is broken :fearful: Kindly fix it to make your readme descriptive and professional :+1: